src/utils/graphql/isFeatureEnabled.tsx

```diff
@@ -1,36 +1,45 @@
 import { graphql, readInlineData } from 'relay-runtime';
 
-import { isFeatureEnabledFragment$key } from '~/generated/isFeatureEnabledFragment.graphql';
+import { isFeatureEnabledFragment$key, Role } from '~/generated/isFeatureEnabledFragment.graphql';
 import isProduction from '~/utils/isProduction';
 
 export enum FeatureFlag {
-  ART_GOBBLERS = 'ART_GOBBLERS',
-  ADMIRE_COMMENT = 'ADMIRE_COMMENT',
   WHITE_RHINO = 'WHITE_RHINO',
   EMAIL = 'EMAIL',
 }
 
 const PROD_FLAGS: Record<FeatureFlag, boolean> = {
-  ART_GOBBLERS: false,
-  ADMIRE_COMMENT: false,
   WHITE_RHINO: false,
   EMAIL: false,
 };
 
 const DEV_FLAGS: Record<FeatureFlag, boolean> = {
-  ART_GOBBLERS: true,
-  ADMIRE_COMMENT: true,
   WHITE_RHINO: true,
   EMAIL: true,
 };
 
 const EMPLOYEE_FLAGS: Record<FeatureFlag, boolean> = {
-  ART_GOBBLERS: false,
-  ADMIRE_COMMENT: true,
   WHITE_RHINO: true,
   EMAIL: true,
 };
 
+const ROLE_FLAGS: Record<Role, Record<FeatureFlag, boolean>> = {
+  ADMIN: {
+    WHITE_RHINO: true,
+    EMAIL: true,
+  },
+  BETA_TESTER: {
+    WHITE_RHINO: true,
+    EMAIL: true,
+  },
+  // In case a new role gets added, how do we want to handle that new role
+  // if the client hasn't been updated to handle the new role.
```

Comment:
lazy solution: deploy backend first then update frontend

longer term: have backend return specific feature flags

Reply:
@Robinnnnn This comment is trying to explain that "the below object is what will be used in the case that the backend deployed something that the frontend doesn't know about"

I'm not literally asking a question, maybe I should change my language here?